test/functional/test-ads-config.js

```diff
@@ -18,6 +18,34 @@ import {adConfig} from '../../ads/_config';
 
 describe('test-ads-config', () => {
 
+  it('should have all ad networks configured', () => {
+    // Get all ad extensions from test env.
+    const extensions = window.ampTestRuntimeConfig.adExtensions;
+    const namingExceptions = {
+      // We recommend 3P ad networks use the same string for filename and ID.
+      // Write exceptions here in alpabetic order.
+      // filename: [ID1, ID2, ... ]
+      adblade: ['adblade', 'industrybrains'],
+      mantis: ['mantis-display', 'mantis-recommend'],
+      weborama: ['weborama-display'],
+    };
+
+    for (let i = 0; i < extensions.length; i++) {
+      const expanded = namingExceptions[extensions[i]];
+      if (expanded) {
+        for (let j = 0; j < expanded.length; j++) {
+          expect(adConfig).to.include.key(expanded[j]);
+        }
+      } else {
+        expect(adConfig).to.include.key(extensions[i]);
+      }
+    }
+
+    // Google's ad networks.
+    expect(adConfig).to.include.key('adsense');
+    expect(adConfig).to.include.key('doubleclick');
```

Comment:
test to include '_ping_'?
Also it is same as `test-integration`. I think it's ok to keep it like now. do you think it's worth to put it in a helper func?